azure-iot-device/azure/iot/device/common/error_map.py

```diff
@@ -0,0 +1,35 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
```

Comment:
Feels like this file, and errors.py could be combined into one module

Reply:
Not used anywhere yet.  This is write-ahead.  I'll need it eventually before the review was scoped down to Paho-only.

---
In reply to: [307876441](https://github.com/Azure/azure-iot-sdk-python-preview/pull/136#discussion_r307876441) [](ancestors = 307876441)